test/Quake/linear_control.qke

```diff
@@ -0,0 +1,78 @@
+// ========================================================================== //
+// Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                 //
+// All rights reserved.                                                       //
+//                                                                            //
+// This source code and the accompanying materials are made available under   //
+// the terms of the Apache License 2.0 which accompanies this distribution.   //
+// ========================================================================== //
+
+// RUN: cudaq-opt --linear-ctrl-form %s | FileCheck %s
+
+// Test the conversion from pruned control form back to linear control form.
+func.func @__nvqpp__mlirgen__function_linear_expr0() {
+  %0 = quake.null_wire
+  %1 = quake.null_wire
+  %2 = quake.null_wire
+  %3 = quake.null_wire
+  %4 = quake.null_wire
+  %5 = quake.null_wire
+  %6 = quake.null_wire
+  %7 = quake.null_wire
+  %8 = quake.null_wire
+  %9 = quake.null_wire
+  %10 = quake.to_ctrl %0 : (!quake.wire) -> !quake.control
```

Comment:
Would it be worth adding another `to_ctrl` here and threading that into one of the existing `quake.x` operations below so that we can test that case, too?